kernel-scala/build.sbt

```diff
@@ -2,6 +2,11 @@
  * Copyright (c) Jupyter Development Team.
  * Distributed under the terms of the Modified BSD License.
  */
+
+import scala.util.Properties
+
+organization := "org.jupyter"
+
 name := "declarativewidgets"
 
 version := "0.1-SNAPSHOT"
```

Comment:
We are going to need a way to set this version based on the version in https://github.com/jupyter-incubator/declarativewidgets/blob/master/nb-extension/python/urth/widgets/ext/_version.py.